tools/library/src/gemm_operation.h

```diff
@@ -566,7 +566,9 @@ class GemmUniversalOperation : public GemmOperationBase<Operator_> {
     operator_args.ldb = int(configuration->ldb);
     operator_args.ldc = int(configuration->ldc);
     operator_args.ldd = int(configuration->ldd);
-    
+
+    operator_args.batch_stride_D = int(configuration->problem_size.m()) * int(configuration->problem_size.n());
```

Comment:
I don't think this change is required. Can you explain what are you trying to achieve here? Also, will this be over-written by run() -> update().

Reply:
Because I tried to implement parallel split K profiling based on GemmUniversal (It's the right way, isn't?), which needs `args.batch_stride_D` to calculate device workspace size in case of parallel split K (https://github.com/NVIDIA/cutlass/blob/856d4db3fbdee6544d9aebd69b89832f569e68bb/include/cutlass/gemm/device/gemm_universal_base.h#L162).